test/test_torch.py

```diff
@@ -6067,8 +6067,8 @@ def test_pow(self, device):
         # [res] torch.pow([res,] x)
 
         # pow has dedicated implementation for different exponents
-        for dtype in torch.testing.get_all_math_dtypes(device):
-
+        dtypes = torch.testing.get_all_math_dtypes(device) + torch.testing.get_all_complex_dtypes()
```

Comment:
get_all_math_dtypes(device) returns complex dtypes as well, so we need to add them separately :D